Library/Formula/aws-sdk-cpp.rb

```diff
@@ -0,0 +1,70 @@
+class AwsSdkCpp < Formula
+  desc "AWS SDK for C++"
+  homepage "https://github.com/aws/aws-sdk-cpp"
+  url "https://github.com/aws/aws-sdk-cpp/archive/0.9.6.tar.gz"
+  sha256 "cf2dad62e50c5c5cff88a4ad959af3aa402aa7e8aa1692bdf807052c0fe4aeb9"
+  head "https://github.com/aws/aws-sdk-cpp.git"
+
+  ALL_CLIENTS = [
```

Comment:
I don't really understand what you mean by that... Take a look at what the implementation in this PR does:

`brew install aws-sdk-cpp` will install all of the clients
`brew install aws-sdk-cpp --with-ec2 --with-s3` will install just the EC2 and S3 clients.

Similarly, the new version could be like:
`brew install aws-sdk-cpp` will install all of the clients
`brew install aws-sdk-cpp --with-compute --with-storage` will install just the clients from the "compute" and "storage" categories